common/lib/driver_connection_provider.ts

```diff
@@ -97,7 +97,7 @@ export class DriverConnectionProvider implements ConnectionProvider {
           JSON.stringify(Object.fromEntries(maskProperties(resultProps)))
       );
 
-      resultTargetClient = driverDialect.connect(hostInfo, resultProps);
+      resultTargetClient = await driverDialect.connect(hostInfo, resultProps);
```

Comment:
Is this required? I think I remembered this causing some issues for one of the drivers.